resources/language/resource.language.es_es/strings.po

```diff
@@ -926,7 +929,7 @@ msgstr "Instalar complemento Up Next"
 
 msgctxt "#30235"
 msgid "[WIP] Synchronize the watched status of the videos with Netflix"
-msgstr "[En desarrollo] Sincronizar el estado de visto de los vídeos con Netflix"
+msgstr "[WIP] Sincronizar el estado de visto de los vídeos con Netflix"
```

Comment:
This is up to you, @CastagnaIT . I would use the 'TEST' word because:
1) It's an universal word and everybody know its meaning. It doesn't matter what the language is.
2) A test function may be a function which is not fully tested or totally finished yet.

But you decide.

Reply:
[WIP] is used in many add-ons. But @CastagnalT decide it.